cme/modules/enum_av.py

```diff
@@ -2,20 +2,16 @@
 # -*- coding: utf-8 -*-
 
 # All credit to @an0n_r0
-# project : https://github.com/tothi/serviceDetector
+# https://github.com/tothi/serviceDetector
+# Module by @mpgn_x64
+# https://twitter.com/mpgn_x64
 
-from impacket.dcerpc.v5 import lsat, lsad
+from impacket.dcerpc.v5 import lsat, lsad, transport
 from impacket.dcerpc.v5.dtypes import NULL, MAXIMUM_ALLOWED, RPC_UNICODE_STRING
-from impacket.dcerpc.v5 import transport
 import pathlib
 
-
 class CMEModule:
-    """
-    Uses LsarLookupNames and NamedPipes to gather information on all endpoint protection solutions installed on the the remote host(s)
-    Module by @mpgn_x64
-    """
```

Comment:
Why remove the Class description?

Reply:
what's its purpose? :)

Reply:
It should tell people looking at the Code what the Module does and who made it :)